src/dotnet/commands/dotnet-complete/ParseCommand.cs

```diff
@@ -24,6 +24,13 @@ public static int Run(string[] args)
 
             Console.WriteLine(result.Diagram());
 
+            var optionValuesToBeForwarded = result.AppliedCommand()
+                                                  .OptionValuesToBeForwarded();
+            if (optionValuesToBeForwarded.Any())
+            {
+                Console.WriteLine("Option values to be forwarded: ");
```

Comment:
@nguerrera  The parse command was something I added to make work on the parser replacement easier and it should be removed or at least made more cohesive with other features intended for developers of the CLI itself, like `--debug` and `dotnet -d`.